src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(int number, String color) {
+        this.number = number;
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public String toString() {
+        return "Number: " + this.getNumber() + ", Color: " + this.getColor();

```

Comment:
```suggestion
        return "Number: " + number + ", Color: " + color;
```
you have direct access